src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/AddAzureVpnClientRevokedCertificateCommand.cs

```diff
@@ -59,10 +59,6 @@ public override void Execute()
         {
 
             base.Execute();
-            if (!this.IsVirtualNetworkGatewayPresent(ResourceGroupName, VirtualNetworkGatewayName))
```

Comment:
Sorry for the inactivity @markcowl 

I removed the code that masks any thrown exceptions. The thrown exceptions provide much more useful information than the fabricated messages.

Yes I have confirmed that this removes the bad error messages in the Issue